gatsby-node.js

```diff
@@ -34,7 +34,7 @@ const createPages = (type, postArray, createPage) => {
   const groupedPosts = groupPostsByUnique(type, postArray)
   // returns {['tag']: [post1, post2], ['tag2']: [post3, post4]}
   Object.entries(groupedPosts).forEach(([typeValue, postGroup], index) => {
-    typeValue = typeValue.split(" ").join("-")
+    typeValue = typeValue.replace(/\.|\s/g, '-').toLowerCase()
```

Comment:
so if you do lowercase on the URL, then you need to do the same in preview-template and blog-post-layout template where the tag links are rendered.

Reply:
Good point. I should read the code it seems.